src/main/java/org/sagebionetworks/web/client/widget/profile/UserProfileEditorWidgetViewImpl.java

```diff
@@ -321,6 +327,10 @@ public void setSynAlert(IsWidget w) {
 	@Override
 	public void setOwnerId(String userId) {
 		_showAccountLevelBadge(accountLevelBadgeContainer.getElement(), userId, propsProvider.getJsniContextProps());
+		UserProfileLinksProps props = UserProfileLinksProps.create(userId);

```

Comment:
New component is configured when the editor is configured with a user ID.  Remember that the ReactComponentDiv cleans up on detach.